core/templates/dev/head/pages/splash/splash.html

```diff
@@ -121,23 +121,21 @@ <h1 class="oppia-splash-h1" translate="I18N_SPLASH_TITLE"></h1>
         </div>
 
         <div class="oppia-splash-button-container">
-          {% if not user_is_logged_in %}
-            <!-- User must complete the registration process. -->
-            <button type="button"
-                    ng-click="onRedirectToLogin('/creator_dashboard?mode=create')"
-                    class="btn oppia-splash-button oppia-splash-button-create oppia-transition-200"
-                    style="left: -webkit-calc(130px + 15%); left: -moz-calc(130px + 15%); left: -o-calc(130px + 15%); left: calc(130px + 15%); bottom: 210px; z-index: 10;"
-                    translate="I18N_ACTION_CREATE_LESSON">
-            </button>
-          {% else %}
-            <button type="button"
-                    ng-click="onClickCreateExplorationButton()"
-                    class="btn oppia-splash-button oppia-splash-button-create oppia-transition-200"
-                    style="left: -webkit-calc(130px + 15%); left: -moz-calc(130px + 15%); left: -o-calc(130px + 15%); left: calc(130px + 15%); bottom: 210px; z-index: 10;"
-                    translate="I18N_ACTION_CREATE_LESSON">
-            </button>
-          {% endif %}
-
+          <!-- User must complete the registration process. -->
```

Comment:
Hi @seanlip, is this comment fit/needed over here ?

Reply:
I think it's OK; it comes from the previous version of the file and is meant to describe the different flow when the user's not logged in.

Reply:
Great, thanks :)